doc/dune-files.rst

```diff
@@ -247,6 +247,9 @@ It contains the following fields:
   <url>)``, ``(documentation <url>)`` are the same (and take precedence over)
   the corresponding global fields. These fields are available since Dune 2.0.
 
+Adding libraries to different projects is done via  ``public_name`` field. See 
+:ref:`library` section for details.
```

Comment:
Do you mean adding libraries to packages?

Reply:
Yeah, you are right.